src/test/java/api/loans/CheckOutByBarcodeTests.java

```diff
@@ -682,6 +724,12 @@ public void canCheckOutUsingUserBarcodeThatContainsSpaces()
     smallAngryPlanet = itemsClient.get(smallAngryPlanet);
 
     assertThat(smallAngryPlanet, hasItemStatus(CHECKED_OUT));
+
+    IndividualResource savedLoan = loansStorageClient.get(response);
```

Comment:
Is this check needed for all of these tests, or could it be sufficient to only check it in the general check out case?

Reply:
Yes, I agrre with you. So, I will add this check only for 'canCheckOutUsingItemAndUserBarcode' test-case.